org.flarerpg.Flare.json

```diff
@@ -0,0 +1,81 @@
+{
+	"app-id": "org.flarerpg.Flare",
+	"rename-desktop-file": "flare.desktop",
+	"rename-icon": "flare",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "1.6",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "flare.sh",
+	"finish-args": [
+		"--socket=wayland",
+		"--socket=x11",
+		"--share=ipc",
+		"--socket=pulseaudio",
+		"--device=dri"
+	],
+	"build-options": {
+		"cflags": "-O2",
+		"cxxflags": "-O2"
+	},
+	"cleanup": [
+		"/app/games",
+		"/lib/debug",
+		"/share/man"
+	],
+	"modules": [
+		{
+			"name": "flare-engine",
+			"buildsystem": "cmake-ninja",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/clintbellanger/flare-engine/archive/v1.0.tar.gz",
+					"sha256": "4bbd4674513b643be6294188904665c53f0ef2912e803212c05e8fd22a44d74d"
+				}
+			],
+			"post-install": [
+				"mkdir /app/bin",
+				"mv /app/games/flare /app/bin"
+			]
+		},
+		{
+			"name": "flare-game",
+			"buildsystem": "cmake-ninja",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/clintbellanger/flare-game/archive/837407a156fb16ec3e4be18d497bae6005d5f946.zip",
+					"sha256": "da06d73d2b3cab3d39fdf947ab886e9aa8068740f562604b953536f3091fc3d8"
+				}
+			],
+			"post-install": [
+				"echo \"flare --mods=fantasycore,empyrean_campaign --data-path=/app/share/games/flare\" > /app/bin/flare.sh",
+				"chmod +x /app/bin/flare.sh",
+				"desktop-file-edit --set-key=Exec --set-value=flare.sh /app/share/applications/flare.desktop"
+				
+			]
+		},
+		{
+			"name": "SDL2",
```

Comment:
It's part of the runtime since a while.

Reply:
Yes, but 2.0.6 causes a segfault. When this (https://github.com/flatpak/freedesktop-sdk-images/pull/84) is merged, SDL can be removed from the Flare manifest.